R/r-to-pmml-define-function.R

```diff
@@ -335,14 +351,22 @@ define_function_get_pmml_str_for_expr <- function(
       get_pmml_str_for_row_access,
       function() {return("")},
       function(cond_expr_id_to_block_expr_ids_mappings) {
-        return(get_pmml_str_for_if_expr(cond_expr_id_to_block_expr_ids_mappings, tokens, orig_func_name, orig_func_param_tokens, is_last_expr, function_scope_variables))
+        return(get_pmml_str_for_if_expr(
+          cond_expr_id_to_block_expr_ids_mappings,
+          tokens,
+          orig_func_name,
+          orig_func_param_tokens,
+          is_last_expr,
+          function_scope_variables
+        ))
       }
     )(expr, tokens, function_scope_variables)
   )
 }
 
 define_function_get_pmml_str_for_token <- function(
-  token, tokens, orig_func_name, orig_func_param_tokens, is_last_expr
```

Comment:
The changes in this function do not contribute to the error and should be moved into a separate commit.